### PR TITLE
[QuickPi][Python][SaveSensors] Save sensors on save file python

### PR DESCRIPTION
**Pull request by NicolasWent**
This pull request allow us to save python sensors inside of a comment of the python code.

The sensors are written in xml and saved at the first line of the program inside of a python comment.

### CODE REVIEW

**Review by mblockelet — APPROVED** [review]

**Comment by NicolasWent** [comment]
I have recreated this request in order to change the way that sensors are saved, they were saved in xml, now they are saved in json in python and in blockly.

In blockly the field "additional" has been added
In python there is a comment at the beginning of the program which contains the variable "additional" in json.

I have also renamed two functions from quickPi, which were called "context.loadPrograms" and "context.savePrograms" to "context.loadAdditional" and "context.saveAdditional", which takes the "additional" variable from python_runner and blockly_runner and the sensors into it.

"Additional" is just an object that contains all additional fields, now it contains only quickpisensors, but later it will also contain the subject and the title of the exercice that the user has set.

File: pemFioi/quickAlgo/python_interface.js [hunk 1]
```diff
@@ -305,9 +336,12 @@ function LogicController(nbTestCases, maxInstructions) {
       }
     }
   };
+  this.getCodeWithSensors = function() {
```
Comment (mblockelet) [inline pemFioi/quickAlgo/python_interface.js]: This should be renamed to something like `getCodeWithAdditional` ou `exportCode`, "sensors" is pretty specific to quick-pi.

**Review by mblockelet — APPROVED** [review]
Just a function to change name but the rest is fine.

File: pemFioi/quickAlgo/python_interface.js (hunk 1 above)
Comment (NicolasWent) [reply pemFioi/quickAlgo/python_interface.js]: I have changed the name of the function to the right one

**Pull request merged**